tutorial/hextof_config.yaml

```diff
@@ -0,0 +1,151 @@
+    timeStamp: 's'
+    energy: 'eV'
+    E: 'eV'
+    kx: '1/A'
+    ky: '1/A'
+
+  channels:
+    timeStamp:
+      format: per_train
+      group_name: "/uncategorised/FLASH.DIAG/TIMINGINFO/TIME1.BUNCH_FIRST_INDEX.1/"
+    pulseId:
+      format: per_electron
+      group_name: "/uncategorised/FLASH.EXP/HEXTOF.DAQ/DLD1/"
+      slice: 2
+    dldPosX:
+      format: per_electron
+      group_name: "/uncategorised/FLASH.EXP/HEXTOF.DAQ/DLD1/"
+      slice: 1
+    dldPosY:
+      format: per_electron
+      group_name: "/uncategorised/FLASH.EXP/HEXTOF.DAQ/DLD1/"
+      slice: 0
+    dldTimeSteps:
+      format: per_electron
+      group_name: "/uncategorised/FLASH.EXP/HEXTOF.DAQ/DLD1/"
+      slice: 3
+    dldAux:
+      format: per_pulse
+      group_name : "/uncategorised/FLASH.EXP/HEXTOF.DAQ/DLD1/"
+      slice: 4
+      dldAuxChannels:
+        sampleBias: 0
+        tofVoltage: 1
+        extractorVoltage: 2
+        extractorCurrent: 3
+        cryoTemperature: 4
+        sampleTemperature: 5
+        crystalVoltage: 6
+        dldTimeBinSize: 15
+    pulserSignAdc: # ADC containing the pulser sign (1: value approx. 35000, 0: 33000)
+      format: per_pulse
+      group_name: "/FL1/Experiment/PG/SIS8300 100MHz ADC/CH6/TD/"
+      #slice: 0
+    monochromatorPhotonEnergy:
+      format: per_train
+      group_name: "/FL1/Beamlines/PG/Monochromator/monochromator photon energy/"
+    gmdBda: # The GMDs can not be read yet...
+      format: per_train
+      group_name: "/FL1/Photon Diagnostic/GMD/Average energy/energy BDA/"
+    #  slice: ":"
+    #gmdTunnel: # The GMDs can not be read yet...
+    #  format: per_pulse
+    #  group_name: "/FL1/Photon Diagnostic/GMD/Pulse resolved energy/energy tunnel/"
+    #  slice: ":"
+    bam: # Here we use the DBC2 BAM as the "normal" one is broken.
+        format: per_pulse
+        group_name: "/uncategorised/FLASH.SDIAG/BAM.DAQ/FL0.DBC2.ARRIVAL_TIME.ABSOLUTE.SA1.COMP/"
+    delayStage:
+      format: per_train
+      group_name: "/zraw/FLASH.SYNC/LASER.LOCK.EXP/F1.PG.OSC/FMC0.MD22.1.ENCODER_POSITION.RD/dGroup/"
+  stream_name_prefixes:
+    pbd: "GMD_DATA_gmd_data"
+    pbd2: "FL2PhotDiag_pbd2_gmd_data"
+    fl1user1: "FLASH1_USER1_stream_2"
+    fl1user2: "FLASH1_USER2_stream_2"
+    fl1user3: "FLASH1_USER3_stream_2"
+    fl2user1: "FLASH2_USER1_stream_2"
+    fl2user2: "FLASH2_USER2_stream_2"
+  beamtime_dir:
+    pg2: "/asap3/flash/gpfs/pg2/"
+    hextof: "/asap3/fs-flash-o/gpfs/hextof/"
+    wespe: "/asap3/fs-flash-o/gpfs/wespe/"
+
+energy:
+  calibration:
+    offset: 4150.0
+    coeffs: [-2.01882455e-05,  1.94714008e-01]
+    Tmat: [[ 1.01040e+06,  1.20000e+02],
+        [ 7.18675e+05,  8.50000e+01],
+        [ 3.82275e+05,  4.50000e+01],
+        [-3.86325e+05, -4.50000e+01]]
+    bvec: [ 3.,  2.,  1., -1.]
+    energy_scale: kinetic
+    axis: -463.3385531514501
+    E0:  -463.3385531514501
+    refid: 3
+  offsets:
+    sampleBias:
+      mode: direct
+      sign: -1
+
+
+metadata:
```

Comment:
This is really bad. Please remove this ingetsor username and password asap.

Reply:
essentially removed from commit history as well

Reply:
I'm  not sure how to remove this from the commit history.
I did not notice this being here. But why was this here in the first place??? these are NOT things for an unencripted config file!

Reply:
currently, scicat didn't roll out for users and we are trying to figure out. till that happened, we are using these details to fetch the metadata.. i think somehow these were left whereever you got the config file from
anyways we are working on a solution that allows us to use desy keycloak to login to the scicat server

Reply:
in the mean time, lets not keep the credentials in the config file, or read them from there. I'm sorry I made the mistake, but this can happen too easily.

Reply:
I agree with you. I will try to come up with a solution.